benchmarks/benchmarks_stats.py

```diff
@@ -0,0 +1,34 @@
+"""Benchmarks suite for stats module."""
+
+import numpy as np
+import xarray as xr
+
+from sgkit import count_call_alleles, count_cohort_alleles
+from sgkit.tests.test_aggregation import get_dataset
+
+
+def time_count_call_alleles() -> None:
+    count_call_alleles(
+        get_dataset(
+            [
+                [[0, 0], [0, 0], [0, 0]],
+                [[0, 0], [0, 0], [0, 1]],
+                [[1, 1], [0, 1], [1, 0]],
+                [[1, 1], [1, 1], [1, 1]],
+            ]
+        )
+    )
+
+
+def time_count_cohort_alleles() -> None:
+    ds = get_dataset(
+        [
+            [[0, 0], [0, 0], [0, 0], [0, 0]],
+            [[0, 0], [0, 0], [0, 1], [0, 1]],
+            [[1, 1], [0, 1], [1, 0], [1, 0]],
+            [[1, 1], [1, 1], [1, 1], [1, 1]],
+        ]
+    )
```

Comment:
These datasets seem rather small. Something 10 or 100 times larger would still run in under a second, and might be more representative of the per-row processing time (since one-off costs would be amortized).

Reply:
Agreed, these are too small to be meaningful. We can use ``simulate_genotype_call_dataset`` to make some fixtures we can use. Probably we want something like samples = [10, 100, 1000] for a fixed 1000 (?) sites.